astropy/io/fits/header.py

```diff
@@ -897,6 +898,18 @@ def get(self, key, default=None):
         except (KeyError, IndexError):
             return default
 
+    def quantity(self, key):
```

Comment:
I'm not sure this is necessary. Why not have the card itself carry a `.quantity` attribute? I suggest to leave this out for now.

Reply:
Right